### PR TITLE
adding multi language function

### PR DESCRIPTION
**Pull request by nikhilkammula**

### CODE REVIEW

File: app/translation/es.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  },
+  {
+    "o": "Recommendations",
+    "t": "Recomendaciones"
+  },
+  {
+    "o": "Spell Check",
+    "t": "Corrector ortográfico"
+  }
+]
```
Comment (GreihMurray) [inline app/translation/es.json:34]: Where did you get the translations in this file? Was it google translate?

**Review by GreihMurray — COMMENTED** [review]
Can you get Irish built into this as well?

File: app/translation/es.json (hunk 1 above)
Comment (nikhilkammula) [reply app/translation/es.json:34]: Yeah they are from google translate

**Comment by GreihMurray** [comment]
Hey Ive actually just seen this but there are gonna be merge conflicts if this gets merged in. Those need fixed before this can be approved

**Pull request closed without merging**